py-polars/polars/internals/frame.py

```diff
@@ -64,8 +64,6 @@
 
 try:
     import pyarrow as pa
-    import pyarrow.compute
```

Comment:
These were needed. I shall add it to the tests

Reply:
Good to add to the tests in that case, because otherwise it 100% looks like it is redundant.

Reply:
Yeap, I did